src/renderer/components/icon/icon.tsx

```diff
@@ -245,6 +255,12 @@ const RawIcon = withTooltip((props: IconProps) => {
   }
 
   if (href) {
+    if (hrefValidation.exec(href) === null) {
+      logger.warn("[ICON]: href prop is unsafe, blocking", { href });
+
+      return null;
```

Comment:
why not show the icon still and just remove the href?

Reply:
Sure